sw/cpp/cheri_sanity/irom.ldscript

```diff
@@ -8,5 +8,6 @@ SECTIONS
 		boot.S.o(.text);
 		boot.cc.o(.text .text.* .rodata .rodata.* .data.rel.ro);
 		end.S.o(.text);
+		. = ALIGN (4);
```

Comment:
```suggestion
		. = ALIGN(4);
```

Reply:
Sorry missed this, fixed with a force push. THE POWA!!!